Gemfile.lock

```diff
@@ -204,16 +204,16 @@ GEM
       bcrypt (~> 3.0)
       devise (> 3.5.2, < 5)
       rails (>= 4.2.0, < 7.2)
-    diff-lcs (1.5.0)
+    diff-lcs (1.5.1)
     digest-crc (0.6.4)
       rake (>= 12.0.0, < 14.0.0)
     docile (1.4.0)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
-    dotenv (2.8.1)
-    dotenv-rails (2.8.1)
-      dotenv (= 2.8.1)
-      railties (>= 3.2)
+    dotenv (3.1.2)
```

Comment:
This is upgraded from `2.8.1` to. `3.1.2`

[V3](https://github.com/bkeepers/dotenv/releases/tag/v3.0.0) had the following breaking changes
- Requires Ruby >= 3.0 and Rails >= 6.1

Following deprecations that will affect us in the future
- `Dotenv::Railtie` is deprecared, we need to use `Dotenv::Rails` instead
- The dotenv-rails gem is now superfluous. It's not technically deprecated yet and will continue to work, but the dotenv